addons/binding/org.openhab.binding.icloud/src/main/java/org/openhab/binding/icloud/discovery/DeviceDiscovery.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.icloud.discovery;
```

Comment:
This should be moved to `org.openhab.binding.icloud.internal.discovery`

Reply:
Done.